data/README.md

```diff
@@ -1,13 +1,60 @@
-# WIP
```

Comment:
Not sure about the directory name `data`. Any better suggestions?

Reply:
"analysis"? 🤷‍♂